sonar-plugin-api/src/main/java/org/sonar/api/measures/Measure.java

```diff
@@ -609,17 +608,20 @@ public String getUrl() {
 
   /**
    * @since 2.14
+   * @deprecated in 6.5 with end of support of Developer cockpit plugin. Always return {@code null}.
    */
   @CheckForNull
+  @Deprecated
   public Integer getPersonId() {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)